docs/source/core_services/drivers/driver_configuration/ecobee_web_driver.rst

```diff
@@ -141,69 +148,110 @@ This configuration works as follows:
 
-Below are the setup instructions.
+.. note::
 
-::
+    This guide assumes the user is in the VOLTTRON_ROOT directory, the VOLTTRON platform has been installed and
+    bootstrapped per the instructions in the VOLTTRON README.
+
+Below are the setup instructions for the Ecobee driver.
+
+
+1. If the platform has not been started:
+
+.. code-block:: Bash
+
+    ./start-volttron
+
+2. Be sure that the environment has been activated - you should see (volttron) next to <user>@<host> in your terminal
+window. To activate an environment, use the following command.
 
-    1. If the platform has not been started:
+.. code-block:: Bash
 
-        ./start-volttron
+    . env/bin/activate
 
-    2. If the environment has not been activated - you should see (volttron) next to <user>@<host> in your terminal window
+3. Install a Master Driver if one is not yet installed
 
-        . env/bin/activate
+.. code-block:: Bash
 
-    4. Install a Master Driver if one is not yet installed
+    python scripts/install-agent.py -s services/core/MasterDriverAgent -c <master driver configuration file>
 
-        python scripts/install-agent.py -s services/core/MasterDriverAgent -c <master driver configuration file>
+4. Load the driver configuration into the configuration store ("vctl config list platform.driver" can be used to show
+   installed configurations)
 
-    5. Load the driver configuration into the configuration store ("vctl config list platform.driver" can be used to show installed configurations)
+.. code-block:: Bash
 
-        vctl config store platform.driver <device topic (mentioned in driver configuration section)> <path to driver configuration>
+    vctl config store platform.driver <device topic (mentioned in driver configuration section)> <path to driver configuration>
 
-    6. Load the driver's registry configuration into the configuration store
+5. Load the driver's registry configuration into the configuration store
 
-        vctl config store platform.driver <registry configuration path from driver configuration> <path to registry configuration file> --csv
+.. code-block:: Bash
 
-    7. Start the master driver
+    vctl config store platform.driver <registry configuration path from driver configuration> <path to registry configuration file> --csv
 
-        vctl start platform.driver
+6. Start the master driver
 
-At this point, the master driver will start, configure the driver agent, and
-data should start to publish on the publish interval. If the authentication code
-provided in the configuration file (as above) is out of date, a new
-authentication code will be obtained by the driver. This will require the user
-enter the pin (found in the volttron logs) into the MyApps section of the Ecobee
-web UI. Failure to do so within 60 seconds will result in the driver being unable
-to get Ecobee data. Instructions on how to enter the pin will be included below.
+.. code-block:: Bash
 
+    vctl start platform.driver
```

Comment:
btw this command will not work as it's not based upon identity at this point...if you included a tag you could do this with -t pd or whatever